### PR TITLE
uncrustify tool for picongpu/PMacc

### PR DESCRIPTION
**Pull request by psychocoderHPC**
add small script to format code with uncrustify and our [style file](https://github.com/ComputationalRadiationPhysics/picongpu/blob/master/doc/picongpu_uncrustify.cfg)

### CODE REVIEW

**Comment by ax3l** [comment]
cool, thank you!

Can you please add a check and a help for `$0` and `$1`?

**Comment by psychocoderHPC** [comment]
`$0` need no check, it is the name of the shell script (including the path) and is always valid.

I'll add a help and a check for `$1`

**Comment by psychocoderHPC** [comment]
add help and check for `$1`

File: src/tools/bin/uncrustifyMyCode [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+if [ "$1" = "-h" ] || [ "$1" = "--help" ] ; then
+    usage 1>&2
+    exit 1
+fi
+
+# check whether the first argument is a file
+if test ! -f "$fileName" ; then
+ echo "\"$fileName\" is not a file" >&2
+ exit 1
```
Comment (ax3l) [inline src/tools/bin/uncrustifyMyCode]: after the first echo, call `usage` before exit

Comment (ax3l) [inline src/tools/bin/uncrustifyMyCode]: pls add consistent spaces (also 4 here)

**Comment by psychocoderHPC** [comment]
- if `$1` is no file echo error message and show help
- fix indention

**Comment by ax3l** [comment]
:+1:

**Pull request merged**